lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -1282,6 +1318,8 @@ public function getColumnName($fieldName)
      *      columnName?: string,
      *      inherited?: class-string,
      *      nullable?: bool,
+     *      insertable?: bool,
```

Comment:
It would be better if we invert these here to `notInsertable` and `notUpdateable`. Then if a user sets `insertable=true` don't set this field at all. Then the code in `BasicEntityPersister` must only test for `isset($fieldMapping['notInsertable`])`

This optimizes the cache size of a serialized ClassMetadata by not storing anything in the 99% default case where insertable=true or updatetable=true.